students/lee_deitesfeld/lesson05/mailroom.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
+import sys
+from operator import itemgetter
+
+main_prompt = "\n".join(("Please choose from below mailroom options:",
+          "1 - Manage Donors and Donations",
+          "2 - Create a Report",
+          "3 - Send letters to all donors",
+          "4 - Quit",
+          ">>> "))
+
+
+sub_prompt = "\n".join(("Please make a selection from the following options",
+                        "1 - List the Names of Donors",
+                        "2 - Add a New Donation",
+                        "3 - Return to Main Menu"
+                        ">>> "))
+
+dic_donors = {"Bob Jones": [63.00, 30.00],
+             "Sue Smith": [50.00],
+             "Joe Grimes": [102.00, 45.00, 70.00],
+             "Andrea Funk": [21.00, 21.00],
+             "Mickey Mouse": [500.00, 250.00, 100.00]}
+
+def menu_selection(prompt, main_dispatch):
+    """Pulls up main menu"""
+    try:
+        while True:
+            response = input(prompt)
+            if main_dispatch[response]() == "exit menu":
+                break
+    except:
```

Comment:
you should catch specific exception here, and also only have it around the line that can throw that exception.